changelog/2.072.0_pre.dd

```diff
@@ -124,30 +124,32 @@ $(HR)
 $(BUGSTITLE Language Changes,
     $(LI $(LNAME2 deprecated_commaexp, Using the result of a comma expression is deprecated.)
 
-    $(P Comma expressions expressions have proven to be a frequent source of confusion, and bug.
-        Using their result will now trigger a deprecation message.)
+    $(P Comma expressions have proven to be a frequent source of confusion, and bugs -
+    see $(LINK2 https://issues.dlang.org/show_bug.cgi?id=2659#c2, here) for examples.)
```

Comment:
The simple reason is it's shorter. I've written a bunch of documentation and it helps. Please let's not argue on something this petty - just (preferably) do it or don't. I've approved the PR already so this is advisory. Thanks.